Source/Renderer/PixelDatatype.js

```diff
@@ -20,6 +20,29 @@ define([
         UNSIGNED_SHORT_5_5_5_1 : WebGLConstants.UNSIGNED_SHORT_5_5_5_1,
         UNSIGNED_SHORT_5_6_5 : WebGLConstants.UNSIGNED_SHORT_5_6_5,
 
+        isPackedDatatype : function(pixelDatatype) {
+            return pixelDatatype === PixelDatatype.UNSIGNED_INT_24_8 ||
+                   pixelDatatype === PixelDatatype.UNSIGNED_SHORT_4_4_4_4 ||
+                   pixelDatatype === PixelDatatype.UNSIGNED_SHORT_5_5_5_1 ||
+                   pixelDatatype === PixelDatatype.UNSIGNED_SHORT_5_6_5;
+        },
+
+        sizeInBytes : function(pixelDatatype) {
+            switch (pixelDatatype) {
+                case PixelDatatype.UNSIGNED_BYTE:
+                    return 1;
+                case PixelDatatype.UNSIGNED_SHORT:
```

Comment:
I like this fall-through `case` style as well; I was going to suggest to use it in the PixelFormat.js changes.  Perhaps it's a good idea.